zebra-chain/src/transaction/auth_digest.rs

```diff
@@ -26,26 +27,13 @@ use proptest_derive::Arbitrary;
 #[cfg_attr(any(test, feature = "proptest-impl"), derive(Arbitrary))]
 pub struct AuthDigest(pub [u8; 32]);
 
-impl AuthDigest {
-    /// Return the hash bytes in big-endian byte-order suitable for printing out byte by byte.
-    ///
-    /// Zebra displays transaction and block hashes in big-endian byte-order,
-    /// following the u256 convention set by Bitcoin and zcashd.
```

Comment:
It would be nice to move these docs to the new `display_order` module instead of removing them.